scripts/jenkins/jobs-obs/openstack-cleanvm.yaml

```diff
@@ -39,6 +40,7 @@
         && ( openstack_project=="Juno" && (image=="SLE_11_SP3"))
         || ( openstack_project=="Liberty" && (image=="SLE_12_SP1" || image=="SLE_12" || image=="openSUSE-Leap-42.1"))
         || ( openstack_project=="Mitaka" && (image=="SLE_12_SP1" || image=="SLE_12_SP2" || image=="openSUSE-Leap-42.1"))
+        || ( openstack_project=="Mitaka" && (image=="SLE_12_SP2" || image=="openSUSE-Leap-42.1" || image=="openSUSE-Leap-42.2"))
```

Comment:
and maybe only test Leap 42.2 instead of also leap 42.1 (which has been broken for ages)?